ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -10226,6 +10232,14 @@ void HqlCppTranslator::buildRecordEcl(BuildCtx & subctx, IHqlExpression * record
 }
 
 
+void HqlCppTranslator::addFormatAttribute(ActivityInstance & instance, WuAttr attr, IHqlExpression * record)
+{
+    //MORE: Better would be to add a section to the workunit and only generate the record formats once, and reference them from here
```

Comment:
HPCC-22805 has been added to come back to this, but I doubt it will be worth it unless there are more reasons to make the change.